lib/src/table/Table.tsx

```diff
@@ -100,6 +100,11 @@ const DxcTableContainer = styled.div<{ margin: TablePropsType["margin"] }>`
     background-color: ${(props) => props.theme.scrollBarTrackColor};
     border-radius: 6px;
   }
+  &:focus {
```

Comment:
Should the table be focused? We should talk to Lara about this.

Reply:
I agree, this should be discussed, if it is not considered a correct approach I think that we have to ensure somehow that the internal information of the table is focusable. Otherwise, keyboard users won't be able to scroll through the table content.